bottom/__init__.py

```diff
@@ -11,42 +13,66 @@
 class Client(event.EventsMixin):
     __conn_cls = connection.Connection
 
-    def __init__(self, host, port, *, encoding='UTF-8', ssl=True, loop=None):
+    def __init__(self, host: str, port: str, *, encoding: str = 'UTF-8',
+                 ssl: bool = True, loop: asyncio.BaseEventLoop = None) -> None:
+        """
+        :param host: The IRC server address to connect to.
```

Comment:
Thanks for writing these!

I'm not a huge fan of this style over the [Google Style](http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html) for docstrings, but I won't turn down contributed docstrings.  There's nothing wrong with this style specifically, I just find the google style a bit easier on the eyes.